napalm/ios/ios.py

```diff
@@ -502,15 +503,38 @@ def commit_config(self, message="", revert_in=None):
 
         If merge operation, perform copy <file> running-config.
         """
-        if revert_in is not None:
-            raise NotImplementedError(
-                "Commit confirm has not been implemented on this platform."
-            )
+        CISCO_TIMER_MIN = 1
+        CISCO_TIMER_MAX = 120
+        ARCHIVE_DISABLED_MESSAGE = (
+            "For Cisco devices, revert_in requires 'archive' feature to be enabled."
+        )
+
         if message:
             raise NotImplementedError(
                 "Commit message not implemented for this platform"
             )
 
+        if revert_in is not None:
+            if not self._check_archive_feature():
+                raise CommitConfirmException(ARCHIVE_DISABLED_MESSAGE)
+            elif not CISCO_TIMER_MIN * 60 <= revert_in <= CISCO_TIMER_MAX * 60:
+                msg = (
+                    "For Cisco devices revert_in  must be between {} and {} seconds. "
+                    "revert_in will be rounded down to the nearest minute."
+                    "Pass revert_in as a multiple of 60".format(
+                        CISCO_TIMER_MIN * 60, CISCO_TIMER_MAX * 60
+                    )
+                )
+                raise CommitConfirmException(msg)
+            else:
+                revert_in = int(revert_in / 60)
```

Comment:
I would call this `revert_in_min` or something that clearly differentiates it from the passed in argument. I say this as it would be easy to miss that the standard `revert_in` argument which is in seconds, has been converted to a different argument which is now in minutes form. Consequently, I think it makes sense to have a different name for this variable.